docs/docs/shapes.mdx

```diff
@@ -113,13 +190,15 @@ Once the page refreshes, we should now have our custom shape on the canvas.
 
 ## Using starter shapes
 
-You can use "starter" shape utils like `BaseBoxShapeUtil` to get regular rectangular shape behavior.
+You can use "starter" shape utils like `BaseBoxShapeUtil` to get regular
+rectangular shape behavior.
 
 > todo
 
 ## Flags
 
-You can use flags like `hideRotateHandle` to hide different parts of the UI when the shape is selected, or else to control different behaviors of the shape.
+You can use flags like `hideRotateHandle` to hide different parts of the UI when
```

Comment:
This file also involves some auto-formatting changes. See https://github.com/tldraw/tldraw/pull/1695

Reply:
Actually, not gonna do that